### DISCUSSION TITLE
AndroRAT

### DISCUSSION THREAD

**Discussion by Baldhead45** [anirudhmalik/xhunter#143] [Q&A]
I'm trying to research to learn how to use and install Android on my phone can anybody help me please

**Comment by lulavc** [comment]
write me.

**Reply by lolyfe4lyfe-source** [reply]
Possibly provide n assist with simplest way to run androrat from my android device?

**Comment by anirudhmalik** [comment]
Closing in favor of v2. See [v2.0-demo](https://github.com/anirudhmalik/xhunter/releases/tag/v2.0-demo) and the repo README. New bugs: open a fresh issue with repro steps. Thanks.

**Comment by lolyfe4lyfe-source** [comment]
S9 this will allow me to use n run such?
